### PR TITLE
chore: breakpoint setup

### PR DESCRIPTION
**Pull request by tborres**
This PR defines breakpoint sizes (based on [bootstrap](https://getbootstrap.com/docs/5.0/layout/breakpoints/)), and introduces utility constants for quick media queries.

Closes [B2IO-4](https://b2io.atlassian.net/browse/B2IO-4?atlOrigin=eyJpIjoiNzk1NjFlYTAxNmY0NDdhZWEwN2FjZTIxZWJmODlhNjEiLCJwIjoiaiJ9) #done

### CODE REVIEW

File: src/breakpoints.ts [hunk 1]
```diff
@@ -0,0 +1,17 @@
+export const bp = {
+  xs: 0,
+  sm: 576,
+  md: 768,
+  lg: 992,
+  xl: 1200,
+  xxl: 1400,
```
Comment (sludemann) [inline src/breakpoints.ts:7]: These don't seem to match the description on the card.  Was that intentional?

**Review by sludemann — APPROVED** [review]
Changes make sense, just one requirements question.

File: src/breakpoints.ts (hunk 1 above)
Comment (tborres) [reply src/breakpoints.ts:7]: Yes(ish). The description on the card was a quick eyeballing of what sizes we had in the BFR designs. This list is based on the breakpoints that bootstrap uses.

Comment (tborres) [reply src/breakpoints.ts:7]: I've updated the card descript to be even more vague but less contradictory.

**Pull request merged**